.prettierrc

```diff
@@ -1,6 +1,9 @@
 {
   "singleQuote": true,
   "semi": false,
-  "trailingComma": "none",
-  "proseWrap": "always"
+  "trailingComma": "es5",
+  "proseWrap": "always",
+  "bracketSpacing": true,
```

Comment:
I'm coming from standard-js, so the original settings are most compatible between the two. Would like to keep the opinionated settings to an absolute minimum.

Reply:
Okay. I will revert that